homeassistant/components/trafikverket_weatherstation/strings.json

```diff
@@ -17,5 +17,53 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "wind_direction_text": {
+        "state": {
+          "east": "East",
+          "north_east": "North east",
+          "east_south_east": "East-south east",
+          "north": "North",
+          "north_north_east": "North-north east",
+          "north_north_west": "North-north west",
+          "north_west": "North west",
+          "south": "South",
+          "south_east": "South east",
+          "south_south_west": "South-south west",
+          "south_west": "South west",
+          "west": "West"
+        }
+      },
+      "precipitation_amountname": {
+        "state": {
+          "error": "Error",
+          "mild_rain": "Mild rain",
+          "moderate_rain": "Moderate rain",
+          "heavy_rain": "Heavy rain",
+          "mild_snow_rain": "Mild rain and snow mixed",
+          "moderate_snow_rain": "Moderate rain and snow mixed",
+          "heavy_snow_rain": "Heavy rain and snow mixed",
+          "mild_snow": "Mild snow",
+          "moderate_snow": "Moderate snow",
+          "heavy_snow": "Heavy snow",
+          "other": "Other",
+          "none": "None",
```

Comment:
Does none need to be translated?

Reply:
I think `None` looks better than `none` as a state of precipitation where there is none.

Reply:
I see, you have the text "none", not the `None` keyword (which will be auto translated).